joomla/joomla-cms
# [4.0] Remove component sidebar from com_content

Pull Request for Issue # .

### Summary of Changes
This component sidebar items for articles are identical to the main nav items. This PR removes the component sidebar.

### Testing Instructions
Navigate through all items under 'Content' and ensure the sidebar is removed and table displaying full width. Check that other list views are uneffected.


### Before
![content-sb1](https://cloud.githubusercontent.com/assets/2803503/26631822/d75e0c44-4604-11e7-9def-1e464194ad36.png)



### After
![content-sb2](https://cloud.githubusercontent.com/assets/2803503/26631789/aa3f3800-4604-11e7-9b81-9be333109329.png)



### Documentation Changes Required
